build.sbt

```diff
@@ -129,7 +126,7 @@ commands += Command.command("metadoc-site") { s =>
     "--clean-target-first",
     "--target",
     "target/metadoc",
-    "example/target/scala-2.12/classes"
+    classDirectory.in(example, Compile).value
```

Comment:
👍 I tried this inside a `test.in(Test) := ...` but got illegal dynamic reference errors.